core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -9250,15 +9279,16 @@ LogicalProject(NAME=[$0], ENAME=[$2])
 ]]>
         </Resource>
     </TestCase>
-    <TestCase name="testProjectCorrelateTranspose">
+    <TestCase name="testProjectCorrelateTransposeWithOver">
         <Resource name="sql">
```

Comment:
Note that `testProjectCorrelateTranspose` in `RelOptRulesTest.xml` is redundant[1][2].

[1] https://github.com/apache/calcite/blob/master/core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml#L9226
[2] https://github.com/apache/calcite/blob/master/core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml#L9253

Reply:
Ah. Somehow there's a duplicate key in RelOptRulesTest.xml. Maybe it's a merge error, I don't know. Can someone add code to DiffRepository to make it fail when reading a .xml file if there are duplicate keys.

Reply:
@julianhyde I would like to do it. : )

Reply:
Opened an issue[(CALCITE-3009](https://issues.apache.org/jira/browse/CALCITE-3009)) for it.